style.css

```diff
@@ -711,3 +735,279 @@ button:active {
   padding: 6px;
   margin-top: 50px;
 }
+
+.nav-links ul li {
+  list-style: none;
+  display: inline;
+  padding: 8px 12px;
+  position: relative;
+  justify-content: space-between;
+  text-decoration: none;
+  min-width: 375px;
+}
+
+@media screen and (min-width: 1500px) {
```

Comment:
- [ ] Would you please use the required Desktop breakpoint in the CSS file `@media screen and (min-width: 768px`. It should not be more than `1440px` as anything above that would be an extra large screen.
- [ ]  Your `about me` cards appear to be aligned vertically. Would you please align them horizontally to match the Figma design  `Hint` Use `Flexbox`

### Your Design

![image](https://user-images.githubusercontent.com/98453979/186848301-f60a4d70-3011-4c02-b222-45e1359589a8.png)

### Figma design

![image](https://user-images.githubusercontent.com/98453979/186848433-d3db960e-e206-48d1-827b-d368ea478c63.png)